src/templates/BillOfLading/BillOfLadingTemplate.tsx

```diff
@@ -223,7 +223,12 @@ const Section1 = (document: BillOfLadingDocument): JSX.Element => {
         </div>
         <div className="w-1/2 border-black border">
           <div className="p-2" style={{ fontSize: "0.8em" }}>
-            Place of Receipt. Applicable only when document used as Multimodal Transport B/L (see clause 1)
+            Place of Receipt.
+            {document.placeOfReceipt ? (
+              <div className="break-all">{document.placeOfReceipt}</div>
+            ) : (
+              " Applicable only when document used as Multimodal Transport B/L (see clause 1)"
+            )}
           </div>
         </div>
```

Comment:
condition the entire div, if `placeOfReceipt` not exist, don't even show the default. is this text `Applicable only when document used as Multimodal Transport B/L (see clause 1)` a must?